src/java.base/share/classes/javax/net/ssl/KeyManagerFactory.java

```diff
@@ -63,15 +63,10 @@ public class KeyManagerFactory {
      *          implementation-specific default if no such property exists.
      */
     @SuppressWarnings("removal")
-    public static final String getDefaultAlgorithm() {
+    public static String getDefaultAlgorithm() {
         String type;
-        type = AccessController.doPrivileged(new PrivilegedAction<>() {
-            @Override
-            public String run() {
-                return Security.getProperty(
-                    "ssl.KeyManagerFactory.algorithm");
-            }
-        });
+        type = AccessController.doPrivileged((PrivilegedAction<String>) () ->
+            Security.getProperty("ssl.KeyManagerFactory.algorithm"));
```

Comment:
We can probably use `sun.security.action.GetPropertyAction::privilegedGetProperty`. This is inside `java.base` so that class is always available.

Reply:
Wasn't there another bug to address this?

Reply:
Perhaps as part of [JDK-6725221](https://bugs.openjdk.java.net/browse/JDK-6725221)?

Reply:
No problem.

Reply:
JDK-6725221 is about obtaining boolean properties, so not an exact match. The suggested change is so easy, I'm going to do it.

Reply:
sun.security.action.GetPropertyAction::privilegedGetProperty doesn't trim the return value. Could this be a problem?

Reply:
Answer is no.